Web3/Classes/ContractABI/ABI/Eth+ABI.swift

```diff
@@ -13,7 +13,7 @@ import Foundation
 public extension Web3.Eth {
     
     /// The struct holding all `abi` methods
-    public var abi: ABI.Type {
```

Comment:
@ypopovych What is the reasoning behind this?

@pixelmatrix Is there a reason to restrict access to this field?

Reply:
It's not a restriction. Everything defined in public extension is public by default. It will show warnings for all public declarations inside